explainers/prioritized-post-task.md

```diff
@@ -108,14 +152,13 @@ support these (see FAQ below).
    directly in response to user input, or updating in-viewport UI state.
 
 2. **user-visible**: User-visible tasks are those that will be visible to the
-   user, but not either not immediately or do not block the user from
-   interacting with the page. These tasks represent either a different
-   importance or timescale as user-blocking tasks.
+   user, but either not immediately or do not block the user from interacting
+   with the page. These tasks represent either a different importance or
+   timescale as user-blocking tasks.
```

Comment:
s/as/from/ ?

Reply:
Oof, yeah that's a bit of an awkward sentence. I changed it slightly.